homeassistant/components/fully_kiosk/sensor.py

```diff
@@ -0,0 +1,115 @@
+"""Fully Kiosk Browser sensor."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import DATA_MEGABYTES, PERCENTAGE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import FullyKioskDataUpdateCoordinator
+from .entity import FullyKioskEntity
+
+SENSORS: tuple[SensorEntityDescription, ...] = (
+    SensorEntityDescription(
+        key="batteryLevel",
+        name="Battery Level",
+        device_class=SensorDeviceClass.BATTERY,
+        native_unit_of_measurement=PERCENTAGE,
+    ),
+    SensorEntityDescription(key="screenOrientation", name="Screen Orientation"),
+    SensorEntityDescription(
+        key="foregroundApp",
+        name="Foreground App",
+    ),
+    SensorEntityDescription(key="currentPage", name="Current Page"),
+    SensorEntityDescription(
+        key="wifiSignalLevel",
```

Comment:
What would be the unit of measurement for this one?

Reply:
I'm not sure, for my tablets it's always been in the 6-9 range. Though thinking about it more I may just drop it as I'm not sure it's actually useful for automation.